package.json

```diff
@@ -57,8 +59,11 @@
     "class-variance-authority": "^0.6.1",
     "clsx": "^1.2.1",
     "connectkit": "^1.5.3",
+    "cookies-next": "^4.2.1",
     "csstype": "^3.0.10",
+    "csv-writer": "^1.6.0",
```

Comment:
can remove some of these when we merge, these are just for the script, which Jeff is going to rewrite